db/seeds.rb

```diff
@@ -5,3 +5,116 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+User.create(first_name: "Anthony", last_name: "Hawkins", email:"s0menarutofan@gmail.com", password:"password", city: "san francisco", about:"I like apples", gender: 'male', confirmed_at: '2017-10-17 02:26:15.130453', admin: false, training: 'less than 1 year', experience: '1-3 years')
```

Comment:
Note: feel free to use iteration for all these User.create... i.e. create a users array with user info and iterate over each element in array to create a user. Also this is a good case for using a gem called Faker to generate random fake words... and to take it further look into using gem FactoryGirl to autogenerate instances of models